WordPress/src/main/res/layout/people_invite_fragment.xml

```diff
@@ -114,7 +114,6 @@
                 android:layout_height="wrap_content"
                 android:layout_marginLeft="1dp"
                 android:layout_marginTop="5dp"
-                android:background="@drawable/invites_border"
                 android:drawableRight="@drawable/arrow"
```

Comment:
With the border image now gone, the arrow looks weird being aligned at the far end of the screen. As mentioned in my [original comment](https://github.com/wordpress-mobile/WordPress-Android/pull/4548#discussion_r80485376), I would suggest having the arrow _right after_ the textview so it would look grouped with the role's text. That effectively means to have the textview being `wrap_content` width instead of `match_parent`.

I would still suggest having the full width be tappable to bring up the role change dialog though so, the click listener will need to be on a parent element as it was originally done before this PR. Hope this makes sense! Thanks!